### PR TITLE
Add OWNERS file

### PR DESCRIPTION
**Pull request by nirarg**
This is done in parallel to the PR in openshift/release: https://github.com/openshift/release/pull/36972
In order to initiate the openshift-bot for reviews/merges in this repository

### CODE REVIEW

**Comment by pkliczewski** [comment]
@lshannon can you please add missing members to the org?

File: workflow-examples/OWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+approvers:
+- lshannon
```
Comment (eloycoto) [inline workflow-examples/OWNERS:2]: - RichardW98
- anludke

Should be here.

Comment (nirarg) [reply workflow-examples/OWNERS:2]: Done

**Review by pkliczewski — APPROVED** [review]

**Pull request merged**